src/main/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetDTO.java

```diff
@@ -8,4 +10,15 @@
 public class VeilederTilordnetDTO {
     AktorId aktorId;
     VeilederId veilederId;
+
+    @JsonCreator
```

Comment:
this constructor is here to support null values for veilederId

Reply:
An alternative to this would be to use `@Data` instead of `@Value`, I think that would work, but then it's not immutable - tradeoff 🤔

Reply:
seems like json deserialization when `veilederId=null` is not working with `@Data`